common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -150,3 +156,24 @@ def test_save_valid_course_invalid_org(self):
                 'edxxx', 'edx'
             )
         )
+
+
+class AdminUnicodeUserTest(TestCase):
```

Comment:
This class name does not really describe the tests it contains, you are only testing below if username is readonly. So maybe a more appropriate name would be AdminUserTest